src/NuGet.Clients/NuGet.CommandLine/Commands/SearchCommand.cs

```diff
@@ -0,0 +1,178 @@
+                    string.Join(" ", Arguments).Trim(), // The arguments are joined with spaces to form a single query string
+                    searchFilter,
+                    skip: 0,
+                    take: Take,
+                    logger,
+                    cancellationToken);
+
+                Console.WriteLine(sourceSeparator);
+                Console.WriteLine($"Source: {source.Name}");
+
+                if (!results.Any())
+                {
+                    Console.WriteLine(packageSeparator);
+                    Console.WriteLine("No results found.");
+                    Console.WriteLine(packageSeparator + "\n");
+                }
+                else
+                {
+                    if (Verbosity == Verbosity.Quiet)
+                    {
+                        System.Console.WriteLine($"Source: {source.Name}");
+                        System.Console.WriteLine(packageSeparator);
+                    }
+                    PrintResults(results);
+                }
+            }
+        }
+
+        private void PrintResults(IEnumerable<IPackageSearchMetadata> results)
+        {
+            int lineSeparatorLength = 20;
+            string packageSeparator = new string('-', lineSeparatorLength);
+
+            foreach (IPackageSearchMetadata result in results)
+            {
+                Console.WriteLine(packageSeparator);
+
+                CultureInfo culture = CultureInfo.CurrentCulture;
+
+                StringBuilder content = new StringBuilder();
+                content.Append($"> {result.Identity.Id} | {result.Identity.Version.ToNormalizedString()}"); // Basic info (Name | Version)
+
+                if (Verbosity != Verbosity.Quiet)
+                {
+                    if (result.DownloadCount != null)
+                    {
+                        string downloads = string.Format(culture, "{0:N}", result.DownloadCount);
+                        content.Append($" | Downloads: {downloads.Substring(0, downloads.Length - 3)}");
+                    }
+                    else
+                    {
+                        content.Append(" | Downloads: N/A");
+                    }
+                }
+
+                System.Console.WriteLine(content.ToString()); // System.Console is used so that output is not suppressed by Verbosity.Quiet
+
+                if (Verbosity != Verbosity.Quiet && result.Description != null)
+                {
+                    string description = result.Description;
+
+                    if (Verbosity == Verbosity.Normal)
+                    {
+                        if (description.Length > 100)
+                        {
+                            description = description.Substring(0, 100) + "...";
+                        }
+                    }
+                    Console.PrintJustified(2, description);
+                }
+            }
+
+            Console.WriteLine(packageSeparator);
+            System.Console.WriteLine();
```

Comment:
What is the difference between `Console` vs `System.Console` here? Just curious.

Reply:
Just using Console overrides the regular System.Console, and uses NuGet.CommandLine.Console. Whenever I want something to print regardless of the verbosity level, I print using System.Console, since Console.WriteLine() does not print anything if the user used `-Verbosity quiet`.